libknet/host.c

```diff
@@ -562,14 +562,43 @@ static void _clear_cbuffers(struct knet_host *host, seq_num_t rx_seq_num)
 	}
 }
 
+static void _reclaim_old_defrag_bufs(struct knet_host *host, seq_num_t seq_num)
+{
+	seq_num_t head, tail; /* circular buffer indexes */
```

Comment:
The comment is wrong, those are the indexes from the possible-valid-sequence-number window, not "real" buffer indexes. Probably an artifact from copying this from _seq_num_lookup